RecoLocalTracker/SiStripZeroSuppression/python/customise_EmulateHybrid.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+## Add the ZS algorithm (in hybrid emulation mode) before repacking, to produce emulated hybrid samples with
+##   cmsDriver --step RAW2DIGI,REPACK:DigiToZS10RawRepack --customiseRecoLocalTracker/SiStripZeroSuppression/customise_EmulateHybrid.py ...
+def customise(process):
```

Comment:
this better have a more specific name to avoid conflict if multiple customise calls are needed
(I couldn't combine this with ```Validation/Performance/TimeMemoryInfo.py```)